render.go

```diff
@@ -3,6 +3,7 @@ package render
 import (
 	"image/color"
 	"image/draw"
+	"log"
```

Comment:
This is a point that I've been planning to also discuss on typesetting#63, but I think it's a bad pattern to log from library code without providing a mechanism for the caller to supply and customize the logger.

I'd prefer to either drop the logging code or change the API so that the user can supply a `*log.Logger` somewhere.

Reply:
That is fair. Removed for now and we can revisit later